ui/components/app/tab-bar/index.scss

```diff
@@ -68,10 +68,6 @@
         margin-inline-end: 16px;
         flex: 0 0 18px;
```

Comment:
Do we still need these flex settings?

Reply:
Yes -- without it, we get the mess again.